.circleci/config.yml

```diff
@@ -140,6 +140,33 @@ jobs:
             ./tmp/cc-test-reporter sum-coverage tmp/codeclimate.*.json -p 4 -o tmp/codeclimate.total.json
             ./tmp/cc-test-reporter upload-coverage -i tmp/codeclimate.total.json
 
+  update_spree_designs_emails:
+    docker:
+      - image: *image
+    working_directory: ~/spree
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "0b:96:15:2d:34:4d:bc:75:d9:fc:7f:1b:de:a5:70:d5"
```

Comment:
Can we load a fingerprint from the env variables?

Reply:
suddenly no, because `.yml` is not evaluating env variables correctly.
We also tried to do it with @caudatecoder and it's also fails.
So, when I try to print it using `run`, it works fine, because it works like a simple bash, but `.yml` interpreting it wrong.

<img width="1372" alt="Screenshot 2019-10-30 at 12 49 04" src="https://user-images.githubusercontent.com/15068752/67856792-a30e0200-fb15-11e9-84d5-08b48e10cef5.png">